spotifyPlayer.py

```diff
@@ -74,3 +74,19 @@ def checkPlayStatus(fileName):
 		threading.Thread(target=checkPlayStatus, args=(currFileName,)).start()
 
 		return True
+
+	def pause(self):
```

Comment:
Why should `pause` and `resume` return anything at all if they just return `true`?